### PR TITLE
Tony css5

### PR DESCRIPTION
**Pull request by tonywied17**
reply likes

### CODE REVIEW

**Review by BarritoN78 — APPROVED** [review]

File: src/app/components/comment/comment.component.ts [hunk 1]
```diff
@@ -12,12 +12,14 @@ import UserMiniDTO from 'src/app/models/UserMiniDTO';
   styleUrls: ['./comment.component.css']
 })
 export class CommentComponent implements OnInit {
+  replyState: boolean = false;
```
Comment (BarritoN78) [inline src/app/components/comment/comment.component.ts:15]: Sonar lint doesn't like this line. Just remove the ": boolean"

Comment (colbyktang) [reply src/app/components/comment/comment.component.ts:15]: Whoa this is a neat comment

Comment (colbyktang) [reply src/app/components/comment/comment.component.ts:15]: I'll grab this bad boy

**Comment by colbyktang** [comment]
I addressed the ESLint

**Review by NewPagodi — APPROVED** [review]
LGTM

**Pull request merged**